Composer/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "projectId": "27ikaq",
```

Comment:
what's this?

Reply:
this is for connection with the cypress dashboard.